adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -150,6 +153,74 @@ object VariantContextConverter {
       case GenotypeAllele.ALT                                => Allele.create(g.getVariant.getAlternateAllele)
     }
   }
+
+  private[adam] def cleanAndMixInSupportedLines(
```

Comment:
Beyond moving this into `VariantContextConverter`, were there any changes to the method? If so, I'll make a second review pass, but for now I'm assuming it is just a move.